scripts/roomba_controller.py

```diff
@@ -0,0 +1,167 @@
+#! /usr/bin/env python
+
+import sys
+import rospy
+from iarc7_msgs.msg import OdometryArray
+import actionlib
+import tf2_ros
+import threading
+from iarc7_motion.msg import QuadMoveGoal, QuadMoveAction
+from iarc7_safety.SafetyClient import SafetyClient
+
+class RoombaRequests(object):
```

Comment:
This is for a single request, no?  Should probably be called `RoombaRequest`

Reply:
fixed